src/Artsy/Router/buildServerApp.tsx

```diff
@@ -57,19 +58,12 @@ export function buildServerApp(config: ServerRouterConfig): Promise<Resolve> {
         )
 
         if (redirect) {
-          resolve({
-            redirect,
-            // TODO: The docs seem to indicate that if there’s a redirect there
-            //       will not be a status.
-            //       https://github.com/4Catalyzer/found#server-side-rendering
-            status,
-          })
+          resolve({ redirect })
           return
         }
 
         const headTags = [<style type="text/css">{MediaStyle}</style>]
-        const matchingMediaQueries =
-          userAgent && matchingMediaQueriesForUserAgent(userAgent)
+        const matchingMediaQueries = userAgent && matchingMediaQueriesForUserAgent(userAgent) // prettier-ignore
```

Comment:
Why was prettier causing an issue here?

Reply:
Sometimes when there's a bunch of variables in sequence and function names are long its slightly easier to read at a glance if things are uniform. Except for this purpose I rarely use `// prettier-ignore`